botfront/server/migrations.js

```diff
@@ -253,6 +253,52 @@ Migrations.add({
         });
     },
 });
+Migrations.add({
+    version: 10,
+    up: async () => {
+        const checkIsTypeButtons = (content) => {
+            // the only type defining key in the response content should be buttons
+            const key = 'buttons';
+            const included = ['image', 'buttons', 'elements', 'custom', 'attachment', 'quick_replies']
```

Comment:
I think a carrousel would pass this test as it also has the buttons key

Reply:
Ok, I have tried it it's fine, the migrations does not mess up the carousels, because the buttons key in carousels is nested :  
```yaml
template_type: genericelements:  - title: teee    subtitle: tt    image_url: ''    buttons:      - title: ww        type: postback        payload: /chitchat.bye```

Reply:
it has a buttons key but the buttons key is nested in an array so it isn't picked up by the test